### PR TITLE
add container list and newbie ssh instructions

### PR DESCRIPTION
**Pull request by britttttk**
#272 

I didn't include the altcoin docker containers.

Also added newbie instructions for ssh with VPS using LunaNode (can possibly combine with one of the other ssh FAQs instead).

### CODE REVIEW

**Comment by pavlenex** [comment]
I really like the SSH guide Jeff made. https://github.com/JeffVandrewJr/patron/blob/master/SSH.md At least we should link to it, the ssh step is important.

File: Troubleshooting.md [hunk 1]
```diff
@@ -14,7 +14,29 @@ Logs can provide an essential piece of information. In the next few paragraphs,
 
 ### 2.1 BTCPay Logs
 
-Since the v1.0.3.8, you can easily access BTCPay Server logs from the front-end. If you are a server admin, go to Server Settings > Logs and open the logs file. If you don't know what a particular error in the logs means, make sure to mention it when troubleshooting.
+Since the v1.0.3.8, you can easily access BTCPay Server logs from the front-end. If you are a server admin, go to **Server Settings > Logs** and open the logs file. If you don't know what a particular error in the logs means, make sure to mention it when troubleshooting.
+
+If you would like more detailed logs and you're using a Docker deployment, you can view logs of specific Docker containers using the command line. See these [instructions to ssh](FAQ/FAQ-ServerSettings.md#how-to-ssh-into-my-btcpay-running-on-vps) into an instance of BTCPay running on a VPS (like LunaNode).
+
+Below is a general list of the container names used for BTCPay. Follow the lightning examples below for step by step log generation instructions.
```
Comment (pavlenex) [inline Troubleshooting.md]: I'd rather just post a log example here than tell them to follow a lightning example. Just post `docker logs generated_btcpayserver_1` and tell people they can change accordingly.

File: Troubleshooting.md [hunk 2]
```diff
@@ -14,7 +14,29 @@ Logs can provide an essential piece of information. In the next few paragraphs,
 
 ### 2.1 BTCPay Logs
 
-Since the v1.0.3.8, you can easily access BTCPay Server logs from the front-end. If you are a server admin, go to Server Settings > Logs and open the logs file. If you don't know what a particular error in the logs means, make sure to mention it when troubleshooting.
+Since the v1.0.3.8, you can easily access BTCPay Server logs from the front-end. If you are a server admin, go to **Server Settings > Logs** and open the logs file. If you don't know what a particular error in the logs means, make sure to mention it when troubleshooting.
+
+If you would like more detailed logs and you're using a Docker deployment, you can view logs of specific Docker containers using the command line. See these [instructions to ssh](FAQ/FAQ-ServerSettings.md#how-to-ssh-into-my-btcpay-running-on-vps) into an instance of BTCPay running on a VPS (like LunaNode).
```
Comment (pavlenex) [inline Troubleshooting.md]: I would not mention LunaNode specifically, remove `(like LunaNode)` as it gives them undeserved advertisement anyway.

File: Troubleshooting.md [hunk 3]
```diff
@@ -14,7 +14,29 @@ Logs can provide an essential piece of information. In the next few paragraphs,
...
+| Nginx | nginx |
+| c-lightning | btcpayserver_clightning_bitcoin |
+| LND | btcpayserver_lnd_bitcoin |
+| RTL | generated_lnd_bitcoin_rtl_1 |
+| LibrePatron | librepatron |
+| Tor | tor-gen |
+| Tor | tor |
+
 
 ### 2.2 Lightning Network Logs
```
Comment (pavlenex) [inline Troubleshooting.md]: Do you mind editing the Lightning Logs with container names rather than container ID's, it's simpler and reduces user-errors.

Comment (britttttk) [reply Troubleshooting.md]: Changed it, what do you think?

**Pull request merged**